src/patternfly/components/Nav/nav.scss

```diff
@@ -154,6 +154,13 @@
   --pf-c-nav__simple-list-link--m-current--FontWeight: var(--pf-global--FontWeight--semi-bold);
   --pf-c-nav__simple-list-link--m-current--BackgroundColor: var(--pf-global--BackgroundColor--150);
 
+  // List item seperator
+  --pf-c-nav__item--m-separator--Height: var(--pf-global--BorderWidth--sm);
+  --pf-c-nav__item--m-separator--BackgroundColor: var(--pf-global--BorderColor--light);
+  --pf-c-nav__item--m-separator--MarginLeft: var(--pf-c-nav__list-link--PaddingLeft);
```

Comment:
Could you check with design about the top and bottom margins between the separator. Its looking extra tight between items with a separator, compared with not. 
<img width="551" alt="Screen Shot 2019-03-20 at 11 28 30 AM" src="https://user-images.githubusercontent.com/20118816/54697098-5dab4000-4b03-11e9-8e2e-2fe7e81f35c4.png">

For example we did this in the options menu:
<img width="144" alt="Screen Shot 2019-03-20 at 11 29 38 AM" src="https://user-images.githubusercontent.com/20118816/54697198-892e2a80-4b03-11e9-96d1-4fa1942c74c8.png">

If its not in the scope of this PR we should create an issue!

Reply:
Yeah let's go ahead and add the same spacers as in the options menu separator. Consistency and also it aligns with the function of the separator - to separate groups of content. It also just looks nicer!

Reply:
@mceledonia , @mcoker updated the margins for the separator which looks really good, but I think this looks odd. If you're fine with it thats great, but if you have any suggestions let us know (maybe adding more padding not sure)..
<img width="699" alt="Screen Shot 2019-03-27 at 5 11 38 PM" src="https://user-images.githubusercontent.com/20118816/55112838-e271fd00-50b3-11e9-9a48-c6991c1fcb2f.png">

Reply:
That does look a bit odd, but is that how I would expect it to work. Since it's only on hover and only when you're before a divider we can be totally fine with it.